.github/CHANGELOG.md

```diff
@@ -64,6 +64,10 @@
 
 <h3>Improvements</h3>
 
+* The `pennylane.plugins` and `pennylane.beta.plugins` folders have been renamed to 
```

Comment:
Do we want to update "Adds a device test suite, located at `pennylane/plugins/tests`"? This is for the current release so think it makes sense.

Reply:
Also noticed that previously there are some links (line 140 and 143) that might be nice to update - I don't think we should change everything in the history, but it would be good for links to work

Reply:
Would be better to do in separate PR? I am trying to be more disciplined with such things :)